packages/xsnap/lib/console-shim.js

```diff
@@ -0,0 +1,17 @@
+const noop = _ => undefined;
+
+const console = {
+  debug: noop,
+  log: noop,
+  info: noop,
+  warn: noop,
+  error: noop,
+  // used by SES
+  group: noop,
+  groupCollapsed: noop,
+  groupEnd: noop,
+  // others from the MDN / whatwg Console API?
+  // trace, dirxml, ...
+};
```

Comment:
This should use the whitelists at https://github.com/Agoric/SES-shim/blob/901153147d02bc395a0f392e482814b2ccbc54a9/packages/ses/src/error/console.js#L36-L71. Move the whitelist to a more reusable place if that would help.

Reply:
Also, we should use our own `VirtualConsole` type from https://github.com/Agoric/SES-shim/blob/901153147d02bc395a0f392e482814b2ccbc54a9/packages/ses/src/error/types.js#L235 which wraps the TypeScript `Console` rather than just using it directly. The reason is that console is only a de facto standard for which that are multiple somewhat inconsistent codifications. By using our `VirtualConsole` wrapper, we're in more control of how this type changes over time.

Reply:
I added "TODO: consider other methods per SES VirtualConsole" with a pointer to https://github.com/Agoric/agoric-sdk/issues/2146

I'm struggling to use SES types at all, let alone `VirtualConsole`.

So I'd like to leave this out of scope of this PR.

Reply:
I agree. I’m working on exporting SES types. Failing that, copies of interface types are compatible.